test/common/remote-process-simulator.js

```diff
@@ -13,10 +14,27 @@ function RemoteProcessSimulator() {
 
   this.control._write = function(command, enc, cb) {
     // Emit commands for validation in tests
-    this.emit('command', command);
+    self.emit('command', command);
```

Comment:
When correcting the conflict mentioned below, don't forget to update this to `'control'`